scripts/sql_script.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE public.products (
+	id uuid NOT_NULL primary key,
```

Comment:
```
CREATE TABLE products (
    id uuid PRIMARY key default uuid_generate_v4(),
```
when u add `default uuid_generate_v4()` during table creation, u won't need to add  id and UUID() into ur script here: 
```
const query = 'INSERT INTO products (**id,** title, description, price) VALUES($1, $2, $3, $4) RETURNING *';
const values = [**UUID()**,body.title, body.description, body.price];